filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -43,69 +43,74 @@ class DrsPathBuilderSpec extends TestKitSuite with AnyFlatSpecLike with Matchers
+      getParent = null,
+      root = s"drs://$bucket/hello/world/with spaces",
+      name = "",
+      getFileName = null,
+      getNameCount = 1,
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "a path with non-ascii",
       path = s"drs://$bucket/hello/world/with non ascii £€",
       normalize = false,
       pathAsString = s"drs://$bucket/hello/world/with non ascii £€",
       pathWithoutScheme = s"$bucket/hello/world/with non ascii £€",
-      parent = s"drs://$bucket/hello/world/",
-      getParent = s"drs://$bucket/hello/world/",
-      root = s"drs://$bucket/",
-      name = "with non ascii £€",
-      getFileName = s"drs://$bucket/with non ascii £€",
-      getNameCount = 3,
-      isAbsolute = true),
+      parent = null,
+      getParent = null,
+      root = s"drs://$bucket/hello/world/with non ascii £€",
+      name = "",
+      getFileName = null,
+      getNameCount = 1,
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "a gs uri path with encoded characters",
       path = s"drs://$bucket/hello/world/encoded%20spaces",
       normalize = false,
       pathAsString = s"drs://$bucket/hello/world/encoded%20spaces",
       pathWithoutScheme = s"$bucket/hello/world/encoded%20spaces",
-      parent = s"drs://$bucket/hello/world/",
-      getParent = s"drs://$bucket/hello/world/",
-      root = s"drs://$bucket/",
-      name = "encoded%20spaces",
-      getFileName = s"drs://$bucket/encoded%20spaces",
-      getNameCount = 3,
-      isAbsolute = true),
+      parent = null,
+      getParent = null,
+      root = s"drs://$bucket/hello/world/encoded%20spaces",
+      name = "",
+      getFileName = null,
+      getNameCount = 1,
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "a file at the top of the bucket",
       path = s"drs://$bucket/hello",
       normalize = false,
       pathAsString = s"drs://$bucket/hello",
       pathWithoutScheme = s"$bucket/hello",
-      parent = s"drs://$bucket/",
-      getParent = s"drs://$bucket/",
-      root = s"drs://$bucket/",
-      name = "hello",
-      getFileName = s"drs://$bucket/hello",
+      parent = null,
+      getParent = null,
+      root = s"drs://$bucket/hello",
+      name = "",
+      getFileName = null,
       getNameCount = 1,
-      isAbsolute = true),
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "a path ending in /",
       path = s"drs://$bucket/hello/world/",
       normalize = false,
-      pathAsString = s"drs://$bucket/hello/world",
-      pathWithoutScheme = s"$bucket/hello/world",
-      parent = s"drs://$bucket/hello/",
-      getParent = s"drs://$bucket/hello/",
-      root = s"drs://$bucket/",
-      name = "world",
-      getFileName = s"drs://$bucket/world",
-      getNameCount = 2,
-      isAbsolute = true),
+      pathAsString = s"drs://$bucket/hello/world/",
+      pathWithoutScheme = s"$bucket/hello/world/",
+      parent = null,
+      getParent = null,
```

Comment:
TOL: Any idea why we need separate `parent` and `getParent` fields? I can't find any specific usages for `getParent` except one, which looks like it could just point to `parent` instead... 🤔

Reply:
- `getParent` is a [Java API](https://github.com/openjdk/jdk/blob/jdk8-b120/jdk/src/share/classes/java/nio/file/Path.java#L137-L160)
- `parent` is a [Better Files API](https://github.com/pathikrit/better-files/blob/v3.9.1/core/src/main/scala/better/files/File.scala#L117-L133)

They have a similar contract but it's not exactly the same (one example, others in this file too): 

https://github.com/broadinstitute/cromwell/blob/d2f506b76fdb55d7396901a0333d76dd52501e87/filesystems/gcs/src/test/scala/cromwell/filesystems/gcs/GcsPathBuilderSpec.scala#L163-L175

Yay `WomFile`/`java.nio.file.Path`/`better.files.File`/etc. differences!